README.md

```diff
@@ -1,30 +1,118 @@
-# DeltaCAT
+![deltacat-header-logo](media/deltacat-logo-alpha.png)
 
-DeltaCAT is a Pythonic Data Catalog powered by Ray.
+DeltaCAT is a portable Data Lakehouse powered by [Ray](https://github.com/ray-project/ray). It lets you define and manage
+fast, scalable, ACID-compliant multi-modal data lakes, and has been used to [successfully manage exabyte-scale enterprise
+data lakes](https://aws.amazon.com/blogs/opensource/amazons-exabyte-scale-migration-from-apache-spark-to-ray-on-amazon-ec2/).
 
-Its data storage model allows you to define and manage fast, scalable,
-ACID-compliant data catalogs through git-like stage/commit APIs, and has been
-used to successfully host exabyte-scale enterprise data lakes.
+It uses the Ray distributed compute framework together with [Apache Arrow](https://github.com/apache/arrow) and
+[Daft](https://github.com/Eventual-Inc/Daft) to efficiently scale common table management tasks, like petabyte-scale
+merge-on-read and copy-on-write operations.
 
-DeltaCAT uses the Ray distributed compute framework together with Apache Arrow
-for common table management tasks, including petabyte-scale
-change-data-capture, data consistency checks, and table repair.
+DeltaCAT provides four high-level components:
+1. **Catalog**: High-level APIs to create, discover, organize, and manage datasets.
+2. **Compute**: Distributed data management jobs to read, write, and optimize datasets.
+3. **Storage**: In-memory and on-disk multi-modal dataset formats based on Apache Arrow.
+4. **Sync**: Synchronize DeltaCAT datasets with other data warehouses and table formats.
 
-## Getting Started
 
-### Install
+## Getting Started
 
+### Apache Iceberg
+#### Installation
+```shell
+pip install deltacat[iceberg]
 ```
-pip install deltacat
-```
 
-### Running Tests
+#### AWS Glue
+##### Initializing a Catalog
+You can initialize a DeltaCAT catalog from any new or existing Iceberg catalog. When using an existing catalog, you can
+immediately read and write any tables it contains. When creating a new catalog, you can immediately start adding and
+managing new tables.
+```python
+import deltacat as dc
+from deltacat import IcebergCatalog
+
 
+# Start by initializing DeltaCAT and registering all Catalogs you'd like to make available to your Ray application.
+# Ray will be initialized automatically via `ray.init()`.
+
```

Comment:
Yes - this likely needs to be refactored to not implicitly always depend on a Ray Actor to track all registered catalogs (since that really only applies when sharing registered catalog info with all nodes in a cluster).